app-modules/theme/src/Filament/Pages/ManageBrandConfigurationSettings.php

```diff
@@ -73,6 +78,14 @@ public function form(Form $form): Form
     {
         return $form
             ->schema([
+                Section::make('Application Name')
+                    ->aside()
+                    ->schema([
+                        TextInput::make('application_name')
+                            ->label(false)
+                            ->dehydrated(fn (?string $state): bool => filled($state))
+                            ->maxLength('255'),
```

Comment:
This should be required and can be dehydrated even if it was just filled with the default. That way on save it actually gets set, which is not a problem.